staff/oihane-unciti/mucho-layout/all-sass/style.css

```diff
@@ -0,0 +1,336 @@
+body {
+  padding-top: 10%;
+  width: 99vw;
+  margin: 0 auto;
+  border: 2px solid;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+.header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  border-bottom: 1px solid gray;
+  padding-bottom: 2vh;
+  top: 0;
+  left: 0;
+  background-color: white;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+}
+
+.header__presentation {
+  display: none;
+  width: 40%;
+}
+
+.menu {
+  list-style: none;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 30%;
+  margin-right: 0.5em;
+}
+
+.menu__li {
+  margin-left: 2%;
+  display: none;
+}
+
+.menu--top {
+  display: none;
+}
+
+.menu__red {
+  list-style: block;
+  padding: 2%;
+}
+
+.logo {
+  padding: 2%;
+  width: 15%;
+}
+
+.nav {
+  height: 10%;
+  width: 100%;
+}
+
+.nav__border {
+  border-bottom: 1px solid lightgray;
+  margin-left: 1%;
+}
+
+.container_img {
+  width: 95%;
+  margin: 0 auto;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+}
+
+.article {
+  width: 95%;
+  margin: 0 auto;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  margin-bottom: 10px;
+}
+
+.article__information {
```

Comment:
In order to have a similar size in this section than in the images you could use min-height and min-width.